src/backend/marsha/deposit/tests/api/depositedfiles/test_delete.py

```diff
@@ -0,0 +1,96 @@
+"""Tests for the deposited files delete API."""
+from django.test import TestCase, override_settings
+
+from marsha.core.factories import (
+    OrganizationAccessFactory,
+    PlaylistAccessFactory,
+    PlaylistFactory,
+)
+from marsha.core.models import ADMINISTRATOR
+from marsha.core.simple_jwt.factories import (
+    InstructorOrAdminLtiTokenFactory,
+    StudentLtiTokenFactory,
+    UserAccessTokenFactory,
+)
+from marsha.core.tests.testing_utils import reload_urlconf
+from marsha.deposit.factories import DepositedFileFactory
+
+
+# We don't enforce arguments documentation in tests
+# pylint: disable=unused-argument
+
+
+@override_settings(DEPOSIT_ENABLED=True)
+class DepositedFileDeleteAPITest(TestCase):
```

Comment:
In all the tests you should test the number of resource in database before and after the delete request.